.scripts/generateDataplaneList.ts

```diff
@@ -22,5 +22,7 @@ const getRushPackages = () => {
     packageFolders.push(path.basename(path.dirname(filePath)));
     packageNames.push(proj.packageName);
   }
+  // Explicitly avoid packaging any packages in event-processor-host
+  packageFolders.push("event-processor-host");
```

Comment:
Is there more generic coding pattern we can create here instead of hardcoding specific package names in our shared scripts?

Reply:
Unfortunately, we don't have any flag in package.json and @ramya-rao-a and @jeremymeng has mentioned that they don't want to remove this folder altogether.

Reply:
This script is used only by track 1 packaging script.

Reply:
Can we not use the type defined in the project.json to handle this filtering?

Reply:
We don't have any flag in package.json to decide filtering. But adding a property called `"publishToNpm"; false` is probably a better generic solution. 

@ramya-rao-a @jeremymeng Do you have any concern in adding this new property in package.json for event-processor-host package?

Reply:
@weshaggard I am discussing with JS team offline to come up with a better solution here.

Reply:
This change is reverted and no longer required since we are deleting samples.